src/librustdoc/clean/utils.rs

```diff
@@ -455,3 +456,10 @@ crate fn find_nearest_parent_module(tcx: TyCtxt<'_>, def_id: DefId) -> Option<De
         None
     }
 }
+
+crate fn has_doc_flag(attrs: Attributes<'_>, flag: Symbol) -> bool {
+    attrs.iter().any(|attr| {
+        attr.has_name(sym::doc)
+            && attr.meta_item_list().map_or(false, |l| list_contains_name(&l, flag))
+    })
+}
```

Comment:
Why is this being added? Is it different from [`Attributes::has_doc_flag`](https://doc.rust-lang.org/nightly/nightly-rustc/rustdoc/clean/types/struct.Attributes.html#method.has_doc_flag)? If so, that should probably be documented so we don't confuse people.

Reply:
One takes the `hir::Attributes` whereas the other takes `clean::Attributes`. I'll add a doc comment.

Reply:
Huh, I think I see. It's unfortunate to have them duplicated :/